x-pack/plugins/infra/public/pages/metrics/hosts/components/unified_search_bar.tsx

```diff
@@ -24,7 +24,8 @@ export const UnifiedSearchBar = ({ dataView }: Props) => {
   const {
     unifiedSearchDateRange,
     unifiedSearchQuery,
-    submitFilterChange,
+    unifiedSearchFilters,
+    onSubmit,
     saveQuery,
     clearSavedQUery,
```

Comment:
Would you mind fixing all these from `clearSavedQUery` to `clearSavedQuery`

Reply:
Thank you for spotting that! Fixed ✅